Sources/NIOHPACK/HPACKHeader.swift

```diff
@@ -260,7 +260,15 @@ public struct HPACKHeaders: ExpressibleByDictionaryLiteral {
             return result
         }
         
-        return result.flatMap { $0.split(separator: ",") }.map { String($0._trimWhitespace()) }
+        return result.flatMap {
+            $0.split(separator: ",")
+        }.map {
+            $0._trimWhitespace()
+        }.filter { // `split(separator:)` drops empty strings, we should too.
+            !$0.isEmpty
+        }.map {
+            String($0)
+        }
```

Comment:
We may need to rewrite this to use lazy iteration now.

Reply:
Yup, looks like a bunch of allocation regressions.